### ISSUE TITLE
Warning: Consider adding a lang attribute - wrongly detected

### ISSUE THREAD

**Issue by saslanyan** [validator/validator#936]
This is the message:
Warning: Consider adding a lang attribute to the html start tag to declare the language of this document.

From line 1, column 16; to line 2, column 6

type html>↩<head>↩  <me

And this is the code of the tested website (agenthelpers.net):

```
<!DOCTYPE html>
--
  | <html lang="en">
  | <head>
  | <!-- set the encoding of your site -->
```

**Comment by nschonni** [comment]
Validating by URL seems to trigger this, but now validating by text input. Do you have any line ending issues possibly?

**Comment by sideshowbarker** [comment]
There’s no `html` element at all in the document that the checker receives from that URL: https://validator.w3.org/nu/?showsource=yes&doc=http://agenthelpers.net/#source

Same if I request it using `curl`:

```
$ curl http://agenthelpers.net/
<!doctype html>
<head>
  <meta charset="utf-8">
  <meta http-equiv="x-ua-compatible" content="ie=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
  <title>StackPath</title>
```

**Comment by saslanyan** [comment]
Thank you

**Comment by holoman0** [comment]
Ok, I changed my CDN firewall settings and now the actual page is being returned. When I use the w3c validator here: 
https://validator.w3.org/nu/?showsource=yes&doc=http://agenthelpers.net/#source 
 the validator properly detects the lang="en" and does not report it. However, the Hexometer version of the validator is still reporting this.

**Comment by sideshowbarker** [comment]
> the Hexometer version of the validator is still reporting this.

I don’t know what the Hexometer version of the validator is, but it’s something I’m involved in maintaining. Whatever it is, I guess it’s just not been updated yet.
